Venmo/src/main/java/com/braintreepayments/api/VenmoClient.java

```diff
@@ -110,7 +111,7 @@ public void createPaymentAuthRequest(@NonNull final FragmentActivity activity,
                 callback.onVenmoPaymentAuthRequest(new VenmoPaymentAuthRequest.Failure(new BraintreeException(
                         "Cannot collect customer data when ECD is disabled. Enable this feature " +
                                 "in the Control Panel to collect this data.")));
-                braintreeClient.sendAnalyticsEvent("pay-with-venmo.app-switch.failed");
+                braintreeClient.sendAnalyticsEvent(VenmoAnalytics.TOKENIZE_FAILED.event);
```

Comment:
```suggestion
                braintreeClient.sendAnalyticsEvent(VenmoAnalytics.TOKENIZE_FAILED);
```
nit: Would it be better to convert `VenmoAnalytics.TOKENIZE_FAILED` to a `String` and remove the `.event` property access?

Reply:
Yeah I'd go companion `const val` to reduce some of the boilerplate. If we were performing a `switch / when` on it i'd keep it an enum, but for `braintreeClient.sendAnalyticsEvent()` a plain string value is equally sufficient.

Reply:
Cool - updated 👍